src/components/App/App.js

```diff
@@ -5,16 +5,12 @@ import {
   Redirect,
   Switch,
 } from 'react-router-dom';
-
```

Comment:
Lots of white space clean up. If you have your imports organized, you shouldn't need padded lines between everything.